core-bundle/tests/Functional/RoutingTest.php

```diff
@@ -1279,7 +1278,7 @@ public function testCorrectPageForUnknownLanguage(): void
         $this->expectDeprecation('Since contao/core-bundle 4.10: Using the "Contao\CoreBundle\Routing\FrontendLoader" class has been deprecated %s.');
 
         Config::set('folderUrl', true);
```

Comment:
I think you have missed this one (also see below).

Reply:
`folderUrl` is not yet deprecated if `legacy_routing` is active I think. If we want to deprecate `legacy_routing` and the accociated config options I can create a seperate PR.

Reply:
You are right, the option is still used. My bad.